src/sentry/static/sentry/app/views/alerts/list/row.tsx

```diff
@@ -30,6 +30,30 @@ import {getIncidentMetricPreset, isIssueAlert} from '../utils';
 import SparkLine from './sparkLine';
 import {TableLayout, TitleAndSparkLine} from './styles';
 
+/**
+ * Retrieve the start/end for showing the graph of the metric
+ * Will show at least 150 and no more than 10,000 data points
+ */
+export const makeRuleDetailsQuery = (
```

Comment:
I know you're just moving it, we might need to add a unit test just for this function, scary

Reply:
I will remove this soon, I don't like it either, probably will just pass an incident id to focus the chart around